package.json

```diff
@@ -2,7 +2,7 @@
   "name": "learning-js",
   "version": "1.0.0",
   "private": true,
-  "workspaces": ["petar", "aleksandra", "nikolina", "david"],
+  "workspaces": ["petar", "aleksandra", "nikolina", "david", "Public/media-hunt"],
   "description": "**Shortcuts and Most used commands in Linux**",
   "main": "index.js",
   "scripts": {
```

Comment:
Please also add the script so you could execute commands. Like `yarn nikolina appname`, but for this to be something like `yarn public moviehunt`